src/lib/swapper/swappers/CowSwapper/getCowSwapMinMax/getCowSwapMinMax.ts

```diff
@@ -32,10 +35,21 @@ export const getCowSwapMinMax = (
     )
   }
 
+  const getMinCowSwapValueUsd = (chainId: CowChainId): BigNumber.Value => {
```

Comment:
As discussed would prefer a IIFE which is what we conventionally use. Obviously stylistic preferences so won't block the PR for it, but in the case of a simple branching like this, this is what we conventionally use.